<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewSynapseForm</class>
 <widget class="QWidget" name="NewSynapseForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget_Source">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>301</width>
     <height>291</height>
    </rect>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QTableWidget" name="tableWidget_Target">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>301</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>301</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="lineEdit_Target">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Target">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>IP address</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ID</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Amount</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Target</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Source</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>50</y>
     <width>291</width>
     <height>191</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>250</y>
     <width>291</width>
     <height>101</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton_Create">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Create</string>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_w">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_fx">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>14</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>W value</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_type">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Excitatory</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Inhibitory</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>301</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="lineEdit_Source">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Source">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>IP address</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ID</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Amount</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_Exit">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>370</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="toolTipDuration">
    <number>1</number>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
